### PR TITLE
Snapshot tests fixes

### PR DESCRIPTION
**Pull request by EgorovEI**
- In this PR `ConnectView` was fixed according to design, added `Layout` enum to be able to differentiate the screens where `ConnectView` is shown
- Snapshot tests for Chat screen were fixed also by adding mocks.
- Also fixed `OperatorChatMessageView` trailing constraint

[MOB-1803](https://glia.atlassian.net/browse/MOB-1803)

[MOB-1803]: https://glia.atlassian.net/browse/MOB-1803?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: GliaWidgets/Component/Connect/ConnectView.swift [hunk 1]
```diff
@@ -37,35 +56,39 @@ final class ConnectView: BaseView {
...
 
     override func setup() {
         super.setup()
-        setState(.none, animated: false)
         accessibilityElements = [operatorView, statusView]
 
         addSubview(stackView)
         stackView.axis = .vertical
-        stackView.spacing = 32
+        stackView.spacing = State.none.chatContentSpacing
```
Comment (gersonnoboa) [inline GliaWidgets/Component/Connect/ConnectView.swift:70]: Seems like a weird name, `State.none.chatContentSpacing`. Don't really have a better one on the top of my head though. I guess it makes sense if there's state `none`, `connecting`, etc. But it's still weird that a state can be `none`.

**Review by gersonnoboa — APPROVED** [review]

File: GliaWidgets/Component/Connect/ConnectView.swift (hunk 1 above)
Comment (EgorovEI) [reply GliaWidgets/Component/Connect/ConnectView.swift:70]: `none` state is used as default/initial one. What do you think about to name it `initial`?

Comment (gersonnoboa) [reply GliaWidgets/Component/Connect/ConnectView.swift:70]: initial sounds nicer indeed

Comment (rasmustautsglia) [reply GliaWidgets/Component/Connect/ConnectView.swift]: You can use instead:

`private lazy var stackView = UIStackView.make(.vertical, spacing: 32)()`

Comment (EgorovEI) [reply GliaWidgets/Component/Connect/ConnectView.swift]: > initial sounds nicer indeed

22b5f39d04f4285c683d277b8b224c6853381dda

Comment (EgorovEI) [reply GliaWidgets/Component/Connect/ConnectView.swift]: > You can use instead:
> 
> `private lazy var stackView = UIStackView.make(.vertical, spacing: 32)()`

46f3f58990eb69716eef07a92e8a7ef23fa1c489

**Review by rasmustautsglia — APPROVED** [review]

**Comment by EgorovEI** [comment]
!squash

**Pull request merged**